setup.py

```diff
@@ -52,6 +52,7 @@
     'flake8-sfs>=0.0.3,<0.1',
     'isort>=4.3.4,<5',
     'pylint>=2.5.3,<3',
+    'flake8-quotes>=3.3.0,<3.4',
```

Comment:
@pvk-developer maybe we should cap it at `<4`? It seems like they will release the 3.4.0 soon.

Reply:
I agree